inspection/data/results.log

```diff
@@ -0,0 +1,70 @@
+{'page_j': 1, 'test': 'equality', 'page_i': 1, 'result': 'pass', 'case': 'DefaultTest'}
```

Comment:
There are a lot of issues with how the algorithm is being processed; the main function is very difficult to debug.  I added this so that it would be easier for the intern @Tracyfcx  to write the print diff function with unittests,  without having to invoke the algorithm's main function.